src/App.js

```diff
@@ -1,25 +1,49 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFoodForm';
+import SearchBar from './components/SearchBar';
+import { Row, Col } from 'antd'
 
 function App() {
+  const [allFoods, setAllFoods] = useState(foods)
+  
+  const createFood = (newFood) => {
+    const updatedFood = [newFood, ...allFoods]
+    setAllFoods(updatedFood)
+  }
+
+  const searchFood = (filterfoods) => {
+    const searchedFood = allFoods.filter(food => 
+    food.name.toLowerCase().includes(filterfoods.toLowerCase()))
+    setAllFoods(searchedFood)
+  }
+
+  
+
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className='headT'>
+        <AddFood createFood={createFood} />
+      </div>
+      
+        <SearchBar filterfoods={searchFood} />
+      
+        
+      <Row>
+        {allFoods.map((food) => {
```

Comment:
Last but not least, for your search bar to work, you now need to map over the `showFoods` instead of the `dishes`:
`{showFoods.map((food) => {`